lib/src/parse/stylesheet.dart

```diff
@@ -494,6 +533,11 @@ abstract class StylesheetParser extends Parser {
         return _disallowedAtRule(start);
       case "supports":
         return supportsRule(start);
+      case "use":
+        _isUseAllowed = wasUseAllowed;
+        if (!root || !_isUseAllowed) _disallowedAtRule(start);
+        if (!_parseUse) _disallowedAtRule(start);
```

Comment:
Might be worth having a more specific error message here since this is a new feature and users might bump into it when trying to migrate from using `@import`.